modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/impl/IgniteRpcServer.java

```diff
@@ -197,6 +191,26 @@ else if (prc.executor() != null) {
                 LOG.warn("A request execution was rejected [sender={} req={} reason={}]", sender, S.toString(message), e.getMessage());
             }
         }
+
+        private @Nullable RpcProcessor<NetworkMessage> getProcessor(Class<?> cls) {
+            RpcProcessor<NetworkMessage> prc = processors.get(cls.getName());
+
+            if (prc != null) {
+                processors.putIfAbsent(cls.getName(), prc);
```

Comment:
Why `putIfAbsent` if it already exists above?

Reply:
Good catch, I messed up